cmd/compgen/cmd/receiver/templates/config_test.go.tmpl

```diff
@@ -28,23 +29,23 @@ custom config for our custom receiver.
 func TestConfig_Unmarshal(t *testing.T) {
 	type fields struct { // Custom struct for the Config passed into Unmarshal() by reference
 		ControllerConfig scraperhelper.ControllerConfig
-		sample           string
+		Sample           string  `mapstructure:"sample"`
```

Comment:
The changes of casing `Sample` and mapstructure data resolved a few of the failing tests that were giving this error `decoding failed due to the following error(s): '' has invalid keys: sample`